README.md

```diff
@@ -50,152 +48,76 @@ are automatically reflected on preliminary.istio.io.
 then as soon as your PR is merged into master, it will be merged into the current release branch.
 
-### Creating a version
+### Creating a major/minor release
 
 Here are the steps necessary to create a new documentation version. Let's assume the current
-version of Istio is 1.3 and you wish to introduce 1.4 which has been under development.
+version of Istio is 1.6 and you wish to introduce 1.7 which has been under development.
 
 #### When Istio source code is branched
 
-The documentation repo pulls content from the Istio source repos for inclusion in the published site.
-When the source repos are branched in preparation for a release, a few changes are needed in the
-documentation repo to track this:
-
-1. Switch to the **master** branch of the istio/istio.io repo and make sure everything is up to date.
-
-1. Edit the file `Makefile.core.mk` and change the `SOURCE_BRANCH_NAME` variable to the
-name of the newly created source branches (in this case `release-1.4`).
-
-1. Edit the file `data/args.yml` and set the `source_branch_name` field to the name of the newly created source
-branches (in this case `release-1.4`).
-
-1. Run `make update_all` in order to retrieve the latest material from the source repositories.
-
-1. Commit the previous edits to your local git repo and push your **master** branch to GitHub.
+Run `make prepare-1.7.0`, and that's it. This will grab the latest material from the new istio source branch.
 
 #### On the day of the release
 
-##### Creating the release branch
-
-The day of a major Istio release, assuming you've previously done the steps from the above section, you need to:
-
-1. Switch to the **master** branch of the istio/istio.io repo and make sure everything is up to date.
-
-1. Edit the file `scripts/build_archive_site.sh` and add the new archive version
-(in this case `release-1.3`) to the `TOBUILD` variable.
-
-1. Edit the file `data/versions.yml`. Set the `preliminary` field to the next Istio release
-(in this case `1.5`) and the `main` field to the current release (in this case `1.4`).
-
-1. Commit the previous edits to your local git repo and push your **master** branch to GitHub.
-
-1. Create a new release branch off of master, named as release-**major**.**minor** (in this case `release-1.4`). There is one
-such branch for every release.
-
-1. Edit the file `data/args.yml`. Set the `preliminary` field to `false`
-and the the `doc_branch_name` field to the name of the release branch (in this case `release-1.4`).
-
-1. Commit the previous edits to your local git repo and push your **release** branch to GitHub.
+1. Run `make release-1.7.0`. This make target will change some variables in `master` and `release-1.6` as needed, and create a new branch `release-1.7` for the new version.
 
-#### Updating istio.io
+    - For a dry run before official release, run `make release-1.7.0-dry-run`, which will only create a new branch `release-1.7-dry-run`, and not touch any other branches.
```

Comment:
This seems out of place in the "#### On the day of the release" section, since a dry run would likely be done a day or so ahead of the release day.